### PR TITLE
Add netlify link

### PR DESCRIPTION
**Pull request by albertcui**

### CODE REVIEW

**Comment by howardchung** [comment]
Did we just want to add their logo to the sponsors on the home page?

File: src/lang/en-US.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
   "app_blog": "Blog",
   "app_translate": "Translate",
   "app_donate": "Donate",
+  "app_netlify": "Deploys by Netlify",
```
Comment (howardchung) [inline src/lang/en-US.json]: Hosted by?

Comment (albertcui) [reply src/lang/en-US.json]: Deploys by Netlify is the text they use on their badges: https://www.netlify.com/press/

**Comment by howardchung** [comment]
Hmm what if we just put their badge in the sponsors section?

**Comment by howardchung** [comment]
The text is overflowing the bullets onto the next line:
https://deploy-preview-2098--opendota-stage.netlify.com/

**Comment by howardchung** [comment]
Fix the overflow issue and we're good to go?

**Pull request merged**